static/app/components/feedback/feedbackOnboarding/sidebar.tsx

```diff
@@ -71,10 +70,9 @@ function FeedbackOnboardingSidebar(props: CommonSidebarProps) {
         options: supportedProjectItems,
       },
     ];
-  }, [allProjects]);
+  }, [projects]);
 
-  const selectedProject = currentProject ?? projects[0] ?? allProjects[0];
```

Comment:
this seems like a good change, based on the strange defaults and values we saw when things were still loading 👍

Reply:
it took me forever to track this one down